src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -94,15 +97,16 @@ public CaseTypeDefinition getCaseType(int version, String caseTypeId) {
 
     @Override
     public CaseTypeDefinition getCaseType(final String caseTypeId) {
-        LOG.debug("retrieving case type definition for case type: {}", caseTypeId);
```

Comment:
Wondering why we needed to delete this log message?

Reply:
Sonar suggestion ,  one of the code smell flagged by sonar